graphql/schemabuilder/function.go

```diff
@@ -111,6 +115,48 @@ func (sb *schemaBuilder) buildFederatedFunction(typ reflect.Type, m *method) (*g
 	return field, nil
 }
 
+func (sb *schemaBuilder) buildShadowObjectFederationFunction(typ reflect.Type, m *method) (*graphql.Field, error) {
```

Comment:
Add a comment to say this build a fieldfunc to `federation` object and the signature of the functions is 
blahblah?

Reply:
Maybe augment the comment to include the shape of the generated function like 
```
This generates a function 
federation.fieldfunc(blahblah)